### PR TITLE
Pal 1063 performance tests fixes

### PR DESCRIPTION
**Pull request by dev930018**
Awaiting merge of PAL-1025

### CODE REVIEW

File: data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/DataSerialiserPrePopTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+@ContextConfiguration(classes = {ApplicationConfiguration.class, TestAsyncConfiguration.class, AkkaSystemConfig.class})
+@EnableAutoConfiguration
+@EntityScan(basePackageClasses = {AuthorisedRequestEntity.class})
+@EnableJpaRepositories(basePackages = {"uk.gov.gchq.palisade.service.data.repository"})
+@ActiveProfiles({"h2test", "test-serialisers"})
+@Disabled
```
Comment (developer01189998819991197253) [inline data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/DataSerialiserPrePopTest.java]: Delete this test if its disabled or failing

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

File: data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/DataSerialiserPrePopTest.java (hunk 1 above)
Comment (developer01189998819991197253) [reply data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/DataSerialiserPrePopTest.java]: Still needs deleting

File: data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/TestSerialiser.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.component.data.service;
+
+import uk.gov.gchq.palisade.data.serialise.SimpleStringSerialiser;
+
+public class TestSerialiser extends SimpleStringSerialiser {
+    public TestSerialiser(final Class<?> domainType) {
+        // Ignored
```
Comment (ac74475) [inline data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/TestSerialiser.java]: What is this class for?

**Review by ac74475 — CHANGES_REQUESTED** [review]
See comments

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by ac74475 — APPROVED** [review]

**Pull request merged**